.gitignore

```diff
@@ -0,0 +1,11 @@
+.idea
```

Comment:
I doubt this hurts anything, but wasn't expected.

Reply:
Yea I'm using webstorm so this is useful to me. It should not affect anything negatively.